csharp/Facebook.Yoga.Desktop.sln

```diff
@@ -0,0 +1,102 @@
+﻿
```

Comment:
Since the Facebook BUCK does not support building Visual Studio solutions, I did not know of a strategy for building the nuget package automatically... Input would be greatly appreciated :)